src/main/java/com/redhat/sast/api/service/StatisticService.java

```diff
@@ -152,4 +153,31 @@ private OshScanStatusDto convertUncollectedScanToDto(OshUncollectedScan uncollec
         dto.setProcessedAt(uncollectedScan.getCreatedAt());
         return dto;
     }
+
+    /**
+     * Gets job activity statistics for the last 24 hours.
+     * Returns hourly data points showing job counts by status.
+     *
+     * @return list of 24 hourly data points with job counts
+     */
+    public List<JobActivityDataPointDto> getJobActivity24h() {
+        List<JobActivityDataPointDto> dataPoints = new ArrayList<>();
+        Instant now = Instant.now();
+
+        for (int i = 23; i >= 0; i--) {
+            Instant hourStart = now.minusSeconds((long) i * 3600).truncatedTo(java.time.temporal.ChronoUnit.HOURS);
+            Instant hourEnd = hourStart.plusSeconds(3600);
```

Comment:
Shouldn't we simply use `Instant.now()`? It guarantees that we will show the latest data for sure.